config-provisioning/src/main/java/com/yahoo/config/provision/exception/OutOfCapacityException.java

```diff
@@ -1,5 +1,5 @@
-// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
-package com.yahoo.config.provision;
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.config.provision.exception;
```

Comment:
Need to update the `provision-service` at the same time

Reply:
Fixed